feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -2,7 +2,9 @@
   <div
     class="cx-product-card"
     [ngClass]="{
-      'cx-product-card-selected': product.selected && !singleDropdown
+      'cx-product-card-selected':
+        productCardOptions.product.selected &&
```

Comment:
This is probably an issue in strict mode (what if product not defined)

Reply:
Done!